app/services/govuk_emails/monitor.rb

```diff
@@ -28,9 +28,9 @@ def response
     def send_undeliverable_alerts
       return unless HostEnv.production?
 
-      AlertManager.capture_message("Unable to deliver mail to #{scheduled_mail.addressee} - ScheduledMailing record #{scheduled_mail.id}")
+      AlertManager.capture_message("Unable to deliver mail to #{scheduled_mail.addressee} - ScheduledMailing record #{scheduled_mail.id}") unless @scheduled_mail.mailer_method == "citizen_start_email"
```

Comment:
given the repetition might be worth extracting to a method 

```ruby
def citizen_start_email?
  @scheduled_mail.mailer_method == "citizen_start_email"
end
```

and use

```ruby
blah unless citizen_start_email?
```

Reply:
👍 have made this refactor